src/components/appNavigation/AppNavigation.js

```diff
@@ -40,6 +48,33 @@ const AppNavigator = createSwitchNavigator(routes, { initialRouteName })
  * @param {AppNavigationProps} props
  */
 const AppNavigation = ({ navigation }: AppNavigationProps) => {
+  const [getNavigation] = usePropsRefs([navigation])
+
+  const onOpened = useCallback(
```

Comment:
is this for signed in users only, right ?

Reply:
yes